packages/api/lambdas/report-executions.js

```diff
@@ -1,48 +1,17 @@
-const get = require('lodash.get');
+'use strict';
 
-const { getSnsEventMessageObject, isSnsEvent } = require('@cumulus/common/sns-event');
-// Temporarily change require while this module resides in the API package
-// const Execution = require('@cumulus/api/models/executions');
-const Execution = require('../models/executions');
+const attr = require('dynamodb-data-types').AttributeValue;
+const { publishSnsMessage } = require('@cumulus/common/aws');
 
-/**
- * Create/update execution record from SNS message.
- *
- * @param {Object} executionRecord - An execution record
- * @returns {Promise}
- */
-async function createUpdateExecutionRecord(executionRecord) {
-  const executionModel = new Execution();
-  return executionModel.create(executionRecord);
-}
+const handler = async (event) => {
+  const records = event.Records.map(attr.unwrap);
 
-/**
- * Filter and map SNS records to get execution records.
- *
- * @param {Object} event - Incoming event from SNS
- * @returns {Array<Object>} - Array of execution messages
- */
-function getReportExecutionMessages(event) {
-  const records = get(event, 'Records', []);
-  return records
-    .filter(isSnsEvent)
```

Comment:
It may be worth re-adding those checks, I'll admit that these `report-executions.js` updates were a quick "get something quick and dirty up for review" set of updates.

Reply:
`isSnsEvent` and `getSnsEventMessageObject` are still being used elsewhere, so I left them in.